### DISCUSSION TITLE
Working project , mother and daughter for mothers day tomorrow for AI

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2488] [Q&A]
Will post work flow

prompt:

mother and daughter photo, A daughter is __jessicaL__, both of them are sitting in a cafe having lunch, B mother is 50 years old

loras loaded, 

JessicaL ( my ai ) weight 1
perfect eyes 1
perfect hands 1
family Lora weight 2.57 ( was downloaded as said could be used for family pics
first image on speed

![2024-03-09_12-16-16_2494](https://github.com/lllyasviel/Fooocus/assets/35565479/d27f17f1-2eb2-449c-8e70-c4103d6319fe)

will add more

**Comment by lgwjames** [comment]
moved image to in paint, masked out left face and hair and dropped down improve detail 
added in additional prompt

50 year old female face, black hair, short,  grey eyes, red lipstick, nose ring, 

awaiting outcome:

**Comment by lgwjames** [comment]
first result
![2024-03-09_12-47-16_8123](https://github.com/lllyasviel/Fooocus/assets/35565479/aaea6db2-fd6c-4679-a78d-ced6636c8c2a)

now re masking and adding black hair on left

**Comment by lgwjames** [comment]
first go :
![2024-03-09_13-31-21_6252](https://github.com/lllyasviel/Fooocus/assets/35565479/f2d8efa7-9ad5-439e-9139-e7cb73f2ca5b)

some more edits to do

**Comment by lgwjames** [comment]
so in paint again, for a few edits on Jessica ( right ) 
added 2 silver necklaces. and a gold chain on mum and blue top

![2024-03-09_17-16-37_2118](https://github.com/lllyasviel/Fooocus/assets/35565479/4c646383-34b9-42c7-bc4a-ff8f8ca39261)


few more clean ups to do

**Comment by lgwjames** [comment]
just see arm is bad now after edits, so going back a few steps

**Comment by lgwjames** [comment]
ok small change, still working, used 2 image prompts

![2024-03-09_19-31-53_2253](https://github.com/lllyasviel/Fooocus/assets/35565479/27b93cd8-e376-4c91-8981-37f5304ebea2)

**Comment by lgwjames** [comment]
update, hand / fingers to fix and skin

![2024-03-09_21-52-34_6261](https://github.com/lllyasviel/Fooocus/assets/35565479/2070c80a-dbd0-4048-b219-74af296ddb87)


any input people?

**Comment by lgwjames** [comment]
![Jessica Leigh](https://github.com/lllyasviel/Fooocus/assets/35565479/d8cb20a1-8564-4c22-8460-0e190206dde5)

**Comment by R4290N** [comment]
![311470335-2070c80a-dbd0-4048-b219-74af296ddb87](https://github.com/lllyasviel/Fooocus/assets/152029155/82e0d4fc-a9cb-44b6-b453-b8c43d2a12dd)
